utils/restUtils.js

```diff
@@ -6,6 +6,7 @@ export const restErrorResponseHandler = async response => {
   switch (response.status) {
     case 200:
     case 201:
+    case 422:
```

Comment:
This looks good here but I am nervous that in the long run we'll have a 422 somewhere that shouldn't be ignored this way. Maybe you could handle this error in the specific service method instead of here? thoughts?

Reply:
taking this specific case out as the default will return the response.
in the future we can add further logic for a 422 case if needed

Reply:
@joneubank